include/nbl/type_traits.h

```diff
@@ -19,7 +19,7 @@ template <bool... Vals>
 using bool_sequence = std::integer_sequence<bool, Vals...>;
 
 template <typename T, typename... Us>
-_INLINE_VAR constexpr bool is_any_of_v = (... || std::is_same<T, Us>::value);
```

Comment:
this is a window specific flag 

https://github.com/microsoft/STL/blob/main/stl/inc/type_traits

Reply:
thx, I think someone just copied it from the STL verbatim for some other `is___v`